templates/cronjob.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: platform
+  name: aws-credentials-updater
+
+---
+
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can you order these two fields in the following order: `apiVersion`, `kind`? This is to follow other existing files and to be consistent with rest of resources in this file.